samples/tfm/tfm_secure_peripheral/README.rst

```diff
@@ -86,9 +99,22 @@ Integrating secure interrupt
 
 If the secure peripheral generates interrupts, complete the following steps to integrate the interrupt with the TF-M interrupt mechanism:
 
-1. Define the interrupt source and handling type in the partition manifest YAML file `tfm_secure_peripheral_partition.yaml`_, located in the :file:`secure_peripheral_partition` directory:
+1. Define the interrupt source and handling type in the partition manifest YAML file `tfm_secure_peripheral_partition.yaml`_, located in the :file:`secure_peripheral_partition` directory.
+   
+   For example, the following IRQ definition configures the TIMER1 peripheral as secure:
+
 
    .. code-block:: yaml
+   
+      "irqs": [
+            {
+                    "source": "TFM_TIMER1_IRQ",
+                    "name": "TFM_TIMER1_IRQ",
+                    "handling": "FLIH"
+            },
+      ]
+   
+   See `TF-M Secure Interrupt Integration`_  for more details regarding the secure interrupts.
 
       "irqs": [
```

Comment:
You haven't removed this "old" code blocks from before the edits. They are now duplicated, and these old lines are breaking the build. Remove lines 119-125.